WooCommerce/Classes/ViewRelated/Orders/Order Creation/OrderStatusSection.swift

```diff
@@ -0,0 +1,88 @@
+import SwiftUI
+import Yosemite
+
+struct OrderStatusSection: View {
+    let geometry: GeometryProxy
+    let dateCreated: Date
+    let orderStatus: OrderStatus
+
+    var body: some View {
+        Divider()
+
+        VStack(alignment: .leading) {
+            Text(dateString)
+                .footnoteStyle()
+
+            HStack {
+                Text(statusBadgeTitle)
+                    .foregroundColor(.black)
+                    .footnoteStyle()
+                    .padding(.horizontal, Layout.StatusBadge.horizontalPadding)
+                    .padding(.vertical, Layout.StatusBadge.verticalPadding)
+                    .background(Color(statusBadgeColor))
+                    .cornerRadius(Layout.StatusBadge.cornerRadius)
+                Spacer()
+                Button(Localization.editButton) {}
+                    .buttonStyle(LinkButtonStyle())
+                    .fixedSize(horizontal: true, vertical: true)
+                    .padding(.trailing, -Layout.linkButtonTrailingPadding) // remove trailing padding to align button title to the side
```

Comment:
This is to offset Style.defaultEdgeInsets from [ButtonStyles](https://github.com/woocommerce/woocommerce-ios/blob/develop/WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI%20Components/ButtonStyles.swift#L70).